src/node_crypto.h

```diff
@@ -298,6 +317,12 @@ class SSLWrap {
   v8::Persistent<v8::Value> sni_context_;
 #endif
 
+#ifdef TLSEXT_TYPE_application_layer_protocol_negotiation
+  const unsigned char* alpn_protos_;
+  unsigned alpn_protos_len_;
+  const unsigned char* selected_alpn_proto_;
```

Comment:
I think you should set these to nullptr / zero here or in the constructor.  I'd do it here.